genai/client.go

```diff
@@ -84,8 +84,8 @@ const defaultMaxOutputTokens = 2048
 func (c *Client) GenerativeModel(name string) *GenerativeModel {
 	return &GenerativeModel{
 		GenerationConfig: GenerationConfig{
-			MaxOutputTokens: defaultMaxOutputTokens,
-			TopK:            3,
+			MaxOutputTokens: Ptr(int32(defaultMaxOutputTokens)),
+			TopK:            Ptr(int32(3)),
```

Comment:
Why is this 3 a constant here? Is there some default value? Now that we have `Ptr`, can we leave it at default for the API?

Reply:
Good news, the model now works without these defaults, so I removed them.